### ISSUE TITLE
[Issue]: RecursionError: maximum recursion depth exceeded while calling a Python object

### ISSUE THREAD

**Issue by bbecausereasonss** [vladmandic/automatic#1446]
### Issue Description

New Venv, (fresh install). All models causing this on text-image inference.

RecursionError: maximum recursion depth exceeded while calling a Python object
18:10:02-921222 ERROR    Exception: maximum recursion depth exceeded while calling a Python object
18:10:02-923222 ERROR    Arguments: args=('task(o3a3i2350a0272j)', 'A cute puppy', '', [], 44, 6, False, False, 1, 4, 7,
                         1, -1.0, -1.0, 0, 0, 0, False, 640, 768, False, 0.7, 2, 'Latent', 0, 0, 0, [], 0, False,
                         'MultiDiffusion', False, True, 1024, 1024, 96, 96, 48, 1, 'None', 2, False, 10, 1, 1, 64, False,
                         False, False, False, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False, 0.4,
                         0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background',
                         0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2,
                         0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2,
                         -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '',
                         '', 'Background', 0.2, -1.0, False, 3072, 192, True, True, True, False, False, 7, 100,
                         'Constant', 0, 'Constant', 0, 4, False, 'x264', 'blend', 10, 0, 0, False, True, True, True,
                         'intermediate', 'animation', <scripts.controlnet_ui.controlnet_ui_group.UiControlNetUnit object
                         at 0x00000182AB6588B0>, False, '', 0, 'Send to Canvas Editor', False, 1, 0.15, False, 'OUT',
                         ['OUT'], 5, 0, 'Bilinear', False, 'Pooling Max', False, 'Lerp', '', '', False, False, None,
                         True, False, False, 'Matrix', 'Horizontal', 'Mask', 'Prompt', '1,1', '0.2', False, False, False,
                         'Attention', False, '0', '0', '0.4', None, False, 0.75, 1, 1, False, False, False, False, False,
                         False, 0, False, '1:1,1:2,1:2', '0:0,0:0,0:1', '0.2,0.8,0.8', 150, 0.2, 1, 1, 1, 1, 1, 1, 1, 1,
                         1, 1, 1, 1, False, False, 'positive', 'comma', 0, False, False, '', 7, '', [], 0, '', [], 0, '',
                         [], True, False, False, False, 0, False, None, None, False, 50, False, 4.0, '', 10.0, 'Linear',
                         3, False, 30.0, True, False, False, 0, 0.0, 'Lanczos', 1, True, 0, 0, 0.001, 75, 0.0, False,
                         True, 'Blur First V1', 0.25, 10, 10, 10, 10, 1, False, '', '', 0.5, 1, False, 5, 'all', 'all',
                         'all', '', '', '', '1', 'none', False, '', '', 'comma', '', True, '', 10.0, 30.0, True, 0.0,
                         'Lanczos', 1, 0, 0, 75, 0.0001, 0.0, False, True, False, 0, 0, 512, 512, False, True, False,
                         False, 0, 1, False, 1, True, True, False, False, ['left-right', 'red-cyan-anaglyph'], 2.5,
                         'polylines_sharp', 0, False, False, False, False, False, False, 'u2net', False, True, False,
                         'Not set', True, True, '', '', '', '', '', 1.3, 'Not set', 'Not set', 1.3, 'Not set', 1.3, 'Not
                         set', 1.3, 1.3, 'Not set', 1.3, 'Not set', 1.3, 'Not set', 1.3, 'Not set', 1.3, 'Not set', 1.3,
                         'Not set', False, 'None') kwargs={}
18:10:02-932222 ERROR    gradio call: RecursionError
╭────────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────╮
│ C:\Users\xxxx\Deep\automatic\modules\call_queue.py:34 in f                                                           │
│                                                                                                                       │
│    33 │   │   │   try:                                                                                                │
│ ❱  34 │   │   │   │   res = func(*args, **kwargs)                                                                     │
│    35 │   │   │   │   progress.record_results(id_task, res)                                                           │
│                                                                                                                       │
│ C:\Users\xxxx\Deep\automatic\modules\txt2img.py:56 in txt2img                                                        │
│                                                                                                                       │
│   55 │   if processed is None:                                                                                        │
│ ❱ 56 │   │   processed = processing.process_images(p)                                                                 │
│   57 │   p.close()                                                                                                    │
│                                                                                                                       │
│                                               ... 980 frames hidden ...                                               │
│                                                                                                                       │
│ C:\Users\xxxx\Deep\automatic\venv\lib\site-packages\torch\nn\modules\module.py:1627 in __setattr__                   │
│                                                                                                                       │
│   1626 │   │   params = self.__dict__.get('_parameters')                                                              │
│ ❱ 1627 │   │   if isinstance(value, Parameter):                                                                       │
│   1628 │   │   │   if params is None:                                                                                 │
│                                                                                                                       │
│ C:\Users\xxxx\Deep\automatic\venv\lib\site-packages\torch\nn\parameter.py:10 in __instancecheck__                    │
│                                                                                                                       │
│     9 │   │   return super().__instancecheck__(instance) or (                                                         │
│ ❱  10 │   │   │   isinstance(instance, torch.Tensor) and getattr(instance, '_is_param', False)                        │
│    11                                                                                                                 │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
RecursionError: maximum recursion depth exceeded while calling a Python object

### Version Platform Description

WIndows 11, All is based on install Python 3.10, 4090

### Acknowledgements

- [X] I have read the above and searched for existing issues

--

Installed Extensions

2023-05-28  09:42 AM    <DIR>          a1111-sd-webui-locon
2023-05-15  06:43 PM    <DIR>          a1111-sd-webui-lycoris
2023-04-10  09:47 AM    <DIR>          Abysz-LAB-Ext
2023-06-11  10:03 PM    <DIR>          adetailer
2023-05-23  11:29 AM    <DIR>          batch-face-swap
2023-06-06  10:19 PM    <DIR>          canvas-zoom
2023-05-28  09:42 AM    <DIR>          deforum-for-automatic1111-webui
2023-04-13  05:05 PM    <DIR>          depthmap2mask
2023-04-29  06:09 PM    <DIR>          DreamArtist-sd-webui-extension
2023-05-08  09:01 PM    <DIR>          infinite-zoom-automatic1111-webui
2023-06-11  09:51 PM    <DIR>          kandinsky-for-automatic1111
2023-04-01  11:12 PM    <DIR>          multi-subject-render
2023-05-23  11:29 AM    <DIR>          multidiffusion-upscaler-for-automatic1111
2023-05-31  02:49 PM    <DIR>          OneButtonPrompt
2023-04-01  11:15 PM    <DIR>          openOutpaint-webUI-extension
2023-04-01  11:15 PM    <DIR>          openpose-editor
2023-04-01  11:15 PM    <DIR>          posex
2023-05-16  05:52 PM    <DIR>          SadTalker
2023-05-28  09:42 AM    <DIR>          sd-canvas-editor
2023-06-11  10:08 PM    <DIR>          SD-CN-Animation
2023-04-01  11:24 PM    <DIR>          sd-dynamic-thresholding
2023-06-11  10:10 PM    <DIR>          sd-model-organizer
2023-06-06  10:19 PM    <DIR>          sd-web-ui-kitchen-theme
2023-04-29  06:07 PM    <DIR>          sd-webui-3d-open-pose-editor
2023-06-01  01:20 PM    <DIR>          sd-webui-aspect-ratio-helper
2023-06-06  10:19 PM    <DIR>          sd-webui-controlnet
2023-06-11  10:04 PM    <DIR>          sd-webui-faceswap
2023-06-16  05:28 PM    <DIR>          sd-webui-inpaint-anything
2023-05-08  09:04 PM    <DIR>          sd-webui-llul
2023-05-28  09:42 AM    <DIR>          sd-webui-lora-block-weight
2023-05-08  09:04 PM    <DIR>          sd-webui-panorama-viewer
2023-06-01  01:21 PM    <DIR>          sd-webui-regional-prompter
2023-06-05  03:32 PM    <DIR>          sd-webui-segment-anything
2023-06-01  01:21 PM    <DIR>          sd-webui-supermerger
2023-06-01  01:21 PM    <DIR>          sd-webui-text2video
2023-06-11  09:59 PM    <DIR>          sd_webui_SAG
2023-04-09  04:18 PM    <DIR>          stable-diffusion-webui
2023-05-08  09:04 PM    <DIR>          stable-diffusion-webui-artists-ui-tweak
2023-05-28  09:42 AM    <DIR>          Stable-Diffusion-Webui-Civitai-Helper
2023-06-01  01:20 PM    <DIR>          stable-diffusion-webui-composable-lora
2023-04-01  11:18 PM    <DIR>          stable-diffusion-webui-conditioning-highres-fix
2023-06-01  01:20 PM    <DIR>          stable-diffusion-webui-depthmap-script
2023-04-01  11:18 PM    <DIR>          stable-diffusion-webui-pixelization
2023-06-05  03:33 PM    <DIR>          stable-diffusion-webui-rembg
2023-04-25  07:21 PM    <DIR>          stable-diffusion-webui-two-shot
2023-04-09  04:03 PM    <DIR>          stable-diffusion-webui-wd14-tagger
2023-04-13  05:05 PM    <DIR>          StylePile
2023-05-08  09:04 PM    <DIR>          ultimate-upscale-for-automatic1111
2023-06-05  03:32 PM    <DIR>          unprompted

**Comment by bbecausereasonss** [comment]
Also have this warning> 18:23:57-391500 WARNING  Torch BF16 test failed: Fallback to FP16 operations

**Comment by brunogcar** [comment]
1st: run with --debug (to get details)
2nd: run with --safe (to disable all user extensions)

post logs with code tag

**Comment by vladmandic** [comment]
by installing all those extensions in parallel, this is almost like you want to have problems.
like @brunogcar run with `--safe` so no extensions other than built-in get loaded and reproduce. if you cant, then its an issue caused by extension (or combination of extensions) and you need to enable one-by-one to see which one is the cause.

and on top of that you're experimenting with bfloat16 without even understanding what it does. yes, i expect it to fail unless you have torch 2.1 nightly - which i have no idea if you have since i have no version info from you.

and per issue template - no issue should be posted without required information, so marking this as invalid.

**Comment by vladmandic** [comment]
closing as invalid. if this is ever cleaned up properly and all required information is provided, it can be reopened.

**Comment by bbecausereasonss** [comment]
> 

Dumb question but where do I insert those params?

**Comment by bbecausereasonss** [comment]
> by installing all those extensions in parallel, this is almost like you want to have problems. like @brunogcar run with `--safe` so no extensions other than built-in get loaded and reproduce. if you cant, then its an issue caused by extension (or combination of extensions) and you need to enable one-by-one to see which one is the cause.
> 
> and on top of that you're experimenting with bfloat16 without even understanding what it does. yes, i expect it to fail unless you have torch 2.1 nightly - which i have no idea if you have since i have no version info from you.
> 
> all-in-all, i'm not sure what you're even trying to do here. simplify.
> 
> and per issue template - no issue should be posted without required information, so marking this as invalid.

Honest question. Can you try replying without being rude? I'm grateful for this repo, and just seeking help.

I can run Automatic1111's repo with a ton of plugins 99% of the time with no issues (I use these plugins). Maybe I need to enable only the individual plugin(s) I need for each session. 

Turning off plugins seems to have fixed the issue so I'm going to determine which one is causing problems. 

--

08:31:39-974710 INFO     Starting SD.Next
08:31:39-980706 INFO     Python 3.10.11 on Windows
08:31:40-081709 INFO     Version: 87490675 Fri Jun 16 16:18:15 2023 -0400
08:31:40-569705 INFO     Latest published version: 5808b7a6eaeb1f4609f4aa5e1e18f921bb5e544e 2023-06-17T12:13:19Z
08:31:40-571706 DEBUG    Setting environment tuning
08:31:40-572708 DEBUG    Torch overrides: cuda=False rocm=False ipex=False diml=False
08:31:40-573710 DEBUG    Torch allowed: cuda=True rocm=True ipex=True diml=True
08:31:40-577710 INFO     nVidia CUDA toolkit detected
08:31:47-772420 INFO     Torch 2.0.1+cu118
08:31:47-787422 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
08:31:47-789421 INFO     Torch detected GPU: NVIDIA GeForce RTX 4090 VRAM 24564 Arch (8, 9) Cores 128
08:31:47-953418 WARNING  Modified files: ['desktop.ini']
08:31:48-004426 DEBUG    Repository update time: Fri Jun 16 16:18:15 2023
08:31:48-005420 DEBUG    Previous setup time: Fri Jun 16 18:20:53 2023
08:31:48-007420 INFO     Disabled extensions-builtin: ['sd-webui-segment-anything']
08:31:48-008420 INFO     Enabled extensions-builtin: ['a1111-sd-webui-lycoris', 'clip-interrogator-ext', 'LDSR', 'Lora',
                         'multidiffusion-upscaler-for-automatic1111', 'ScuNET', 'sd-dynamic-thresholding',
                         'sd-extension-aesthetic-scorer', 'sd-extension-steps-animation', 'sd-extension-system-info',
                         'sd-webui-agent-scheduler', 'sd-webui-controlnet', 'sd-webui-model-converter', 'seed_travel',
                         'stable-diffusion-webui-images-browser', 'stable-diffusion-webui-rembg', 'SwinIR']
08:31:48-023420 DEBUG    Latest extensions time: Fri Jun 16 18:19:16 2023
08:31:48-024419 DEBUG    Timestamps: version:1686946695 setup:1686954053 extension:1686953956
08:31:48-025419 INFO     No changes detected: Quick launch active
08:31:48-026429 INFO     Running in safe mode without user extensions
08:31:48-061421 INFO     Extension preload: 0.0s C:\Users\xxxx\Deep\automatic\extensions-builtin
08:31:48-105420 DEBUG    Memory used: 0.34 total: 63.91 Collected 0
08:31:48-106420 DEBUG    Starting module: <module 'webui' from 'C:\\Users\\xxxx\\Deep\\automatic\\webui.py'>
08:31:48-107420 INFO     Server arguments: ['--debug', '--safe']
08:31:48-117420 DEBUG    Loading Torch
08:32:08-597830 DEBUG    Loading Gradio
08:32:12-896911 DEBUG    Loading Modules
No module 'xformers'. Proceeding without it.
08:32:20-670312 INFO     Libraries loaded
08:32:20-671312 DEBUG    Entering start sequence
08:32:20-826452 DEBUG    Version: {'app': 'sd.next', 'updated': '2023-06-16', 'hash': '87490675', 'url':
                         'https://github.com/vladmandic/automatic.git/tree/master'}
08:32:20-828452 INFO     Using data path: C:\Users\xxxx\Deep\automatic
08:32:20-831452 DEBUG    Event loop: <_WindowsSelectorEventLoop running=False closed=False debug=False>
08:32:20-832452 DEBUG    Entering initialize
08:32:20-983011 INFO     Available VAEs: C:\Users\xxxx\Deep\automatic\models\VAE 8
08:32:26-572686 INFO     Available models: C:\Users\xxxx\Deep\automatic\models\Stable-diffusion 164
08:32:27-172430 DEBUG    Loading scripts
08:32:31-820044 INFO     ControlNet v1.1.224
ControlNet preprocessor location: C:\Users\xxxx\Deep\automatic\extensions-builtin\sd-webui-controlnet\annotator\downloads
08:32:32-919251 INFO     ControlNet v1.1.224
Image Browser: ImageReward is not installed, cannot be used.
08:32:35-620515 DEBUG    Scripts load: ['a1111-sd-webui-lycoris:2.199s', 'clip-interrogator-ext:0.375s', 'LDSR:0.053s',
                         'Lora:0.215s', 'sd-extension-aesthetic-scorer:0.065s', 'sd-webui-agent-scheduler:0.843s',
                         'sd-webui-controlnet:1.827s', 'seed_travel:0.165s',
                         'stable-diffusion-webui-images-browser:0.235s', 'stable-diffusion-webui-rembg:2.141s',
                         'SwinIR:0.051s', 'ScuNET:0.06s']
08:32:36-066110 INFO     Loading UI theme: name=aliabid94/new-theme style=Auto
08:32:36-096110 DEBUG    Creating UI
08:32:37-299232 DEBUG    Script: 0.39s ui_tabs
                         C:\Users\xxxx\Deep\automatic\extensions-builtin\stable-diffusion-webui-images-browser\scripts\i
                         mage_browser.py
08:32:37-307234 DEBUG    Extensions list loaded: C:\Users\xxxx\Deep\automatic\html\extensions.json
Running on local URL:  http://127.0.0.1:7861

To create a public link, set `share=True` in `launch()`.
08:32:39-331993 INFO     Local URL: http://127.0.0.1:7861/
08:32:39-333987 DEBUG    Gradio registered functions: 1587
08:32:39-334989 INFO     Initializing middleware
08:32:39-352992 DEBUG    Creating API
08:32:39-463988 INFO     [AgentScheduler] Task queue is empty
08:32:39-464991 INFO     [AgentScheduler] Registering APIs
08:32:39-616582 DEBUG    Scripts setup: ['Tiled Diffusion:0.021s', 'ControlNet:0.011s', 'Alternative:0.008s']
08:32:39-617580 DEBUG    Scripts components: []
08:32:39-618580 DEBUG    Model metadata: C:\Users\xxxx\Deep\automatic\metadata.json no changes
08:32:39-843839 DEBUG    gc: collected=16950 device=cuda {'ram': {'used': 1.03, 'total': 63.91}, 'gpu': {'used': 1.52,
                         'total': 23.99}, 'retries': 0, 'oom': 0}
08:32:39-846840 DEBUG    Select checkpoint: duchaitenanime_V40.safetensors [c0cbd34fc5]
08:32:40-057838 DEBUG    gc: collected=124 device=cuda {'ram': {'used': 1.03, 'total': 63.91}, 'gpu': {'used': 1.52,
                         'total': 23.99}, 'retries': 0, 'oom': 0}
08:32:40-058840 DEBUG    Load model weights: existing=False
                         target=C:\Users\xxxx\Deep\automatic\models\Stable-diffusion\duchaitenanime_V40.safetensors
                         info=None
Loading weights: C:\Users\xxxx\Deep\automatic\models\Stable-diffusion\duchaitenanime_V40.safetensors ━━━━━ 0.0/8… -:--:…
                                                                                                            GB
08:32:40-719388 DEBUG    Load model:
                         name=C:\Users\xxxx\Deep\automatic\models\Stable-diffusion\duchaitenanime_V40.safetensors
                         dict=True
08:32:40-720389 DEBUG    Verifying Torch settings
08:32:41-183500 WARNING  Torch BF16 test failed: Fallback to FP16 operations
08:32:41-221495 DEBUG    Torch FP16 test passed
08:32:41-222497 DEBUG    Desired Torch parameters: dtype=BF16 no-half=False no-half-vae=False upscast=False
08:32:41-223498 INFO     Setting Torch parameters: dtype=torch.float16 vae=torch.float16 unet=torch.float16
08:32:41-224497 DEBUG    Torch default device: cuda
08:32:41-225495 DEBUG    Model dict loaded: {'ram': {'used': 1.1, 'total': 63.91}, 'gpu': {'used': 1.58, 'total': 23.99},
                         'retries': 0, 'oom': 0}
08:32:41-242496 DEBUG    Model config loaded: {'ram': {'used': 1.1, 'total': 63.91}, 'gpu': {'used': 1.58, 'total':
                         23.99}, 'retries': 0, 'oom': 0}
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 859.52 M params.
08:32:41-689066 DEBUG    Model created from config: C:\Users\xxxx\Deep\automatic\configs\v1-inference.yaml
08:32:41-691066 DEBUG    Model weights loading: {'ram': {'used': 2.06, 'total': 63.91}, 'gpu': {'used': 1.58, 'total':
                         23.99}, 'retries': 0, 'oom': 0}
08:32:42-848869 DEBUG    Model weights loaded: {'ram': {'used': 7.1, 'total': 63.91}, 'gpu': {'used': 1.58, 'total':
                         23.99}, 'retries': 0, 'oom': 0}
08:32:43-313958 DEBUG    Model weights moved: {'ram': {'used': 5.11, 'total': 63.91}, 'gpu': {'used': 3.61, 'total':
                         23.99}, 'retries': 0, 'oom': 0}
08:32:43-320959 INFO     Applying scaled dot product cross attention optimization
08:32:45-483699 INFO     Embeddings: loaded=39 skipped=46
08:32:45-490697 INFO     Model loaded in 5.4s (load=0.7s config=0.5s create=0.4s apply=0.7s vae=0.5s move=0.5s
                         embeddings=2.2s)
08:32:45-714247 DEBUG    gc: collected=12 device=cuda {'ram': {'used': 5.13, 'total': 63.91}, 'gpu': {'used': 3.61,
                         'total': 23.99}, 'retries': 0, 'oom': 0}
08:32:45-715246 INFO     Model load finished: {'ram': {'used': 5.13, 'total': 63.91}, 'gpu': {'used': 3.61, 'total':
                         23.99}, 'retries': 0, 'oom': 0} cached=0
08:32:46-025245 INFO     Startup time: 57.9s (torch=20.5s gradio=4.3s libraries=7.8s vae=0.3s models=5.6s codeformer=0.4s
                         gfpgan=0.2s scripts=8.5s upscalers=0.1s onchange=0.4s ui=2.9s launch=0.3s app-started=0.3s
                         checkpoint=6.4s)
08:33:53-607059 DEBUG    gc: collected=927 device=cuda {'ram': {'used': 1.15, 'total': 63.91}, 'gpu': {'used': 3.61,
                         'total': 23.99}, 'retries': 0, 'oom': 0}
08:33:53-609055 DEBUG    txt2img: id_task=task(kpl391vc4czwg02)|prompt=little
                         frog|negative_prompt=|prompt_styles=[]|steps=10|sampler_index=6|restore_faces=False|tiling=False
                         |n_iter=1|batch_size=1|cfg_scale=7|clip_skip=1|seed=-1.0|subseed=-1.0|subseed_strength=0|seed_re
                         size_from_h=0|seed_resize_from_w=0|seed_enable_extras=False|height=512|width=512|enable_hr=False
                         |denoising_strength=0.7|hr_scale=2|hr_upscaler=Latent|hr_second_pass_steps=0|hr_resize_x=0|hr_re
                         size_y=0|override_settings_texts=[]args=(0, False, 'MultiDiffusion', False, True, 1024, 1024,
                         96, 96, 48, 1, 'None', 2, False, 10, 1, 1, 64, False, False, False, False, False, 0.4, 0.4, 0.2,
                         0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2,
                         -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '',
                         '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False,
                         0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '',
                         'Background', 0.2, -1.0, False, 0.4, 0.4, 0.2, 0.2, '', '', 'Background', 0.2, -1.0, False,
                         3072, 192, True, True, True, False, False, 7, 100, 'Constant', 0, 'Constant', 0, 4, False,
                         'x264', 'blend', 10, 0, 0, False, True, True, True, 'intermediate', 'animation',
                         <scripts.controlnet_ui.controlnet_ui_group.UiControlNetUnit object at 0x000001AD13791C00>,
                         False, False, 'positive', 'comma', 0, False, False, '', 7, '', [], 0, '', [], 0, '', [], True,
                         False, False, False, 0, False, None, None, False, 50, False, 4.0, '', 10.0, 'Linear', 3, False,
                         30.0, True, False, False, 0, 0.0, 'Lanczos', 1, True, 0, 0, 0.001, 75, 0.0, False, True)
08:33:53-616058 DEBUG    Appplying token merging: current=0 target=0.5
08:33:53-622058 DEBUG    Script process: ['Tiled Diffusion:0.0s', 'Tiled VAE:0.0s', 'Dynamic Thresholding (CFG Scale
                         Fix):0.0s', 'Steps animation:0.0s', 'ControlNet:0.0s']
08:33:53-623061 DEBUG    Script before-process-batch: ['Tiled Diffusion:0.0s', 'Tiled VAE:0.0s', 'Dynamic Thresholding
                         (CFG Scale Fix):0.0s', 'Steps animation:0.0s', 'ControlNet:0.0s']
08:33:53-625057 DEBUG    Script process-batch: ['Tiled Diffusion:0.0s', 'Tiled VAE:0.0s', 'Dynamic Thresholding (CFG
                         Scale Fix):0.0s', 'Steps animation:0.0s', 'ControlNet:0.0s']
Progress 2.3it/s ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00 0:00:04
08:33:59-305834 DEBUG    Script postprocess-batch: ['Tiled Diffusion:0.0s', 'Tiled VAE:0.0s', 'Dynamic Thresholding (CFG
                         Scale Fix):0.0s', 'Steps animation:0.0s', 'ControlNet:0.0s']
08:33:59-310833 DEBUG    Script postprocess-image: ['Tiled Diffusion:0.0s', 'Tiled VAE:0.0s', 'Dynamic Thresholding (CFG
                         Scale Fix):0.0s', 'Steps animation:0.0s', 'ControlNet:0.0s']
08:33:59-312833 DEBUG    Saving image: PNG
                         C:\Users\xxxx\Dropbox\Inbox\Media\AIGENS\outputs\txt2img-images\00000-3833597692-little
                         frog.png (512, 512)
08:33:59-642402 DEBUG    gc: collected=280 device=cuda {'ram': {'used': 3.4, 'total': 63.91}, 'gpu': {'used': 3.82,
                         'total': 23.99}, 'retries': 0, 'oom': 0}
08:33:59-645402 DEBUG    Script postprocess: ['Tiled Diffusion:0.0s', 'Tiled VAE:0.0s', 'Dynamic Thresholding (CFG Scale
                         Fix):0.0s', 'Steps animation:0.0s', 'ControlNet:0.0s']
08:33:59-648402 DEBUG    Appplying token merging: current=0.5 target=0
08:33:59-652405 DEBUG    Processed: 1 Memory: {'ram': {'used': 3.4, 'total': 63.91}, 'gpu': {'used': 3.82, 'total':
                         23.99}, 'retries': 0, 'oom': 0} txt
08:33:59-897082 DEBUG    gc: collected=236 device=cuda {'ram': {'used': 3.4, 'total': 63.91}, 'gpu': {'used': 3.82,
                         'total': 23.99}, 'retries': 0, 'oom': 0}
08:34:00-079080 DEBUG    Server alive: True Memory used: 3.4 total: 63.91

**Comment by bbecausereasonss** [comment]
> by installing all those extensions in parallel, this is almost like you want to have problems. like @brunogcar run with `--safe` so no extensions other than built-in get loaded and reproduce. if you cant, then its an issue caused by extension (or combination of extensions) and you need to enable one-by-one to see which one is the cause.
> 
> and on top of that you're experimenting with bfloat16 without even understanding what it does. yes, i expect it to fail unless you have torch 2.1 nightly - which i have no idea if you have since i have no version info from you.
> 
> all-in-all, i'm not sure what you're even trying to do here. simplify.
> 
> and per issue template - no issue should be posted without required information, so marking this as invalid.

Honest question. Can you try replying without being rude? I'm grateful for this repo, and just seeking help.

I can run Automatic1111's repo with a ton of plugins 99% of the time with no issues (I use these plugins). Maybe I need to enable only the individual plugin(s) I need for each session. 

Thanks for the help.

**Comment by vladmandic** [comment]
sorry for appearing rude, but i review quite a few issues daily and this is by far the worst config i've seen in a long time.
and you have several extensions that are unmaintained by their authors and known to be broken.
and on top of that you said "fresh install", but then report issues with blfoat16 which is highly experimental and definitely not something i'd consider "fresh install".

**Comment by brunogcar** [comment]
dude, i dont think he was being rude,  there are templates in  which most of the issues simply dont follow, you included

reporting a bug is more than welcome  to any project, but first you need to troubleshoot and provide useful info,  that is not just like, hey i have this problem, figure out my problem and fix it for me

lets take you case,  LOTS of exensions, you didnt provide info regarding your plataform, versions, hardware etc at first

while he is improving the SD he tries to bug fix, reply to posts, etc now imagine having to stop and try to  figure out whats is going on with out needed info, several  times a day, everyday just cause ppl dont follow basic instructions and do  minimun search,  that is not just frustrating, that is waste of time

so  figure out which witch extension is broken, create a new issue under [Extension] template is that is just related to vlads fork (with all needed info) and works with a1111, but if its broken on both create an issue in that extension repo, that isnt that hard to optimize his time and get the best possible outcome with issue fixed

**Comment by bbecausereasonss** [comment]
Can you please explain how this is even turned on? I never selected this option. 

12:31:02-901452 WARNING  Torch BF16 test failed: Fallback to FP16 operations

**Comment by vladmandic** [comment]
it looks like you did since test does not run unless bf16 is selected in settings and there is no code that would auto-enable it.
